content/post/Dabbling_with_react_@_React_India_2019.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Dabbling with React @ React India 2019: A Tweet story"
+slug = "reactindia2019-vipulgupta2048"
+image = "images/react1.png"
+date = "2019-10-07T10:10:00"
+description = "My experience of India's first React conf, React India 2019"
+disableComments = false
+author = "Vipul Gupta"
+categories = ["conferences"]
+tags = ['React', 'GraphQL', 'javascript',  'vipulgupta2048', 'community', 'FOSS', 'Redux', 'India', 'React-Native', 'conferences']
++++
+
+Lush green trees, monsoon weather, soul cooling winds coming from the tumoultous waters of the beach where sounds of waves splashing over the dry sand can be heard by people walking by. Now, just imagine a tech conference on a piece of software that powers almost half of the web today at the place that I just described. How could you stop yourself? I couldn't. Here's a brief twitter story of my adventures at React India.
+
+React India was the first ever international, community-led beach conference that provided a platform for developers to share, discuss their insights and experiences with React and React Native. The three-day conference was the first of its kind in India broken into workshops and conference days with community talks from 26th September - 28th September 2019
```

Comment:
```suggestion
React India was the first-ever international, community-led beach conference that provided a platform for developers to share, discuss their insights and experiences with React and React Native. The three-day conference was the first of its kind in India broken into workshops and conference days with community talks from 26th September - 28th September 2019
```

Shouldn't it have a hypen?

Reply:
It should, I just don't like doing it. Will get it done.